### ISSUE TITLE
Update composer.json please

### ISSUE THREAD

**Issue by Amadeco** [staempfli/magento2-module-widget-extra-fields#5]
Good afternoon,

First at all, thank you for your work and the disposal.

Could you update your composer.json please,
the ligne require php –7.0 block our update to the last version of Magento.

Maybe it will be better to look for magento framework requirement.

`
"require": {
    "magento/framework": ">=101.0.0"
}
`

I am looking forward for you answer,

Thank you,

Ilan PARMENTIER

**Comment by mhauri** [comment]
Thank you  @mageho I will have a look at it.

**Comment by Amadeco** [comment]
Thank you for the fix, I will give a try.

**Comment by mhauri** [comment]
Tell me if it works, so I will make a proper release then.

**Comment by Amadeco** [comment]
Good evening @mhauri ,

Unfortunatly, it doesn't work, even with this platform requirement.

```
[InvalidArgumentException]                                                                                                        
  Package staempfli/magento2-module-widget-extra-fields at version  has a PHP requirement incompatible with your PHP version (7.1.16)
```

Only way to install it, was to edit composer.json to emulate php platform
"
    "config": {
    	"platform": {
	    	"php": "7.0" 
	    }
    },
"
But it will cause trouble against other component.

Note, I tried php 7.1 and that was the same problem.

I think these php version requirement seems to be more accurate: 
~7.0.0|~7.1.0|~7.2.0
 
For now, until it is fixed, I used this composer command : 
`composer --ignore-platform-reqs require staempfli/magento2-module-widget-extra-fields`

Thank you

**Comment by mhauri** [comment]
Did you tired it with the "dev-develop" version? Can you tell me wich Magento Version you use and maybe share your composer.json?

**Comment by Amadeco** [comment]
Good morning @mhauri,

Only stable version.
Magento 2.2.4

Extract from our composer.json

> {
    "name": "magento/project-community-edition",
    "description": "eCommerce Platform for Growth (Community Edition)",
    "type": "project",
    "version": "2.2.4",
    "license": [
        "OSL-3.0",
        "AFL-3.0"
    ],
    "require": {
        "magento/product-community-edition": "2.2.4",
        "boldcommerce/magento2-ordercomments": "^1.3",
        "composer/composer": "1.4.1",
        "ethanyehuda/magento2-cronjobmanager": "^1.2",
        "daanbeverdam/checkout-block-provider": "^1.0",
        "faonni/module-breadcrumbs": "^2.2",
        "fooman/tcpdf": "^6.2",
        "fooman/pdfcore-m2": "^9.6",
        "fooman/pdfcustomiser-m2": "^8.0",
        "fooman/emailattachments-m2": "^2.1",
        "Imaginaerum/magento2-language-fr-fr": "*",
        "mageplaza/magento-2-social-login": "*",
        "mageworx/module-seosuiteultimate": "^2.14",
        "mageworx/module-downloads": "^2.4",
        "mageworx/module-advancedproductoptions": "^2.9",
        "mirasvit/module-core": "*",
        "mirasvit/module-kb": "*",
        "mirasvit/module-helpdesk": "*",
        "mirasvit/module-rma": "*",
        "mirasvit/module-cache-warmer": "*",
        "mirasvit/module-reports": "*",
        "mirasvit/module-email": "*",
        "mirasvit/module-cataloglabel": "*",
        "magefan/module-login-as-customer": "^2.1",
        "magehost/performance-dashboard": "^1.13",
        "magepal/magento2-preview-checkout-success-page": "*",
        "magepal/magento2-checkout-success-misc-script": "*",
        "magepal/magento2-googletagmanager": "*",
        "magepal/magento2-customshippingrate": "*",
        "magepal/magento2-customeraccountlinksmanager": "^1.1",
        "owebia/magento2-module-advanced-shipping-setting": "^2.1",
        "pmclain/module-stripe": "^1.1",
        "rubic/magento2-module-clean-checkout": "^1.3",
        "stripe/stripe-php": "5.8.0",
        "smile/elasticsuite": "^2.5",
        "smile/module-elasticsuite-rating": "^2.1",
        "smile/module-elasticsuite-cms-search": "^2.1",
        "smile/elasticsuite-for-retailer": "^1.4",
        "magekey/module-bestseller-widget": "^2.0",
        "mageworx/module-ordereditor": "^3.2",
        "owebia/magento2-module-adv-ship-unlimited-carriers-addon": "1.0.1",
        "webfixit/opcache": "^1.0",
        "trustedshops/trustedshops": "^1.0",
        "dmatthew/magento2-attribute-description": "^0.2.0",
        "mirasvit/module-feed": "*",
        "experius/module-pagenotfound": "^1.2",
        "staempfli/magento2-module-widget-extra-fields": "^1.0"
    },
    "require-dev": {
        "phpunit/phpunit": "~6.2.0",
        "squizlabs/php_codesniffer": "3.2.2",
        "phpmd/phpmd": "@stable",
        "pdepend/pdepend": "2.5.2",
        "friendsofphp/php-cs-fixer": "~2.2.1",
        "lusitanian/oauth": "~0.8.10",
        "sebastian/phpcpd": "2.0.4"
    },
    "autoload": {
        "psr-4": {
            "Magento\\Framework\\": "lib/internal/Magento/Framework/",
            "Magento\\Setup\\": "setup/src/Magento/Setup/",
            "Magento\\": "app/code/Magento/"
        },
        "psr-0": {
            "": [
                "app/code/"
            ]
        },
        "files": [
            "app/etc/NonComposerComponentRegistration.php"
        ],
        "exclude-from-classmap": [
            "**/dev/**",
            "**/update/**",
            "**/Test/**"
        ]
    },
    "autoload-dev": {
        "psr-4": {
            "Magento\\Sniffs\\": "dev/tests/static/framework/Magento/Sniffs/",
            "Magento\\Tools\\": "dev/tools/Magento/Tools/",
            "Magento\\Tools\\Sanity\\": "dev/build/publication/sanity/Magento/Tools/Sanity/",
            "Magento\\TestFramework\\Inspection\\": "dev/tests/static/framework/Magento/TestFramework/Inspection/",
            "Magento\\TestFramework\\Utility\\": "dev/tests/static/framework/Magento/TestFramework/Utility/"
        }
    },
    "minimum-stability": "stable"

**Comment by mhauri** [comment]
Hello @mageho it is fixed in release 1.0.1

**Comment by Amadeco** [comment]
Thank you
